server/controllers/profile.js

```diff
@@ -0,0 +1,102 @@
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /profile/create
+// @desc create a profile
+// @access Private
+exports.createProfile = asyncHandler(async (req, res, next) => {
+    const {
+        userId,
+        firstName,
+        lastName,
+        description,
+        gender,
+        address,
+        birthDate,
+        email,
+        phoneNumber,
+        isAvailable,
+        availability
+    } = req.body
+    
+    const profile = await Profile.create({
```

Comment:
You can check first whether the user already has an existing profile to avoid duplication or conflict.